bin/release-list/README.md

```diff
@@ -0,0 +1,31 @@
+# Release list
+
+Builds a page that lists all Metabase releases for docs: `docs/releases.md`.
+
+## Requirements
+
+This script requires:
+
+- [Babashka](https://github.com/babashka/babashka)
+- [GitHub's CLI (gh)](https://cli.github.com/)
+
+## Generate the list of releases
+
+From this directory, run:
+
+```
+bb -m release-list.main
+```
+
+## Run tests
+
+From this directory, run:
+
+```
+bb test:bb
+```
+
+
+## Editing the page
+
+Edit the file: [./resources/releases.md](./resources/releases.md).
```

Comment:
```suggestion
## Editing the page

Edit the template: [./resources/releases.md](./resources/releases.md).
```

Not sure if this change makes sense to you, but it's more explicit. I'd maybe even call this page `releases-template.md` even.